package.json

```diff
@@ -51,10 +51,11 @@
     "babel-loader": "^6.2.1",
     "babel-plugin-transform-remove-strict-mode": "0.0.2",
     "babel-preset-es2015": "^6.22.0",
+    "babel-preset-2017": "0.0.0",
     "babel-preset-react": "^6.22.0",
     "enzyme": "^3.0.0",
     "enzyme-adapter-react-15": "^1.0.5",
-    "eslint": "^3.1.0",
+    "eslint": "^4.17.0",
```

Comment:
There seems to be a lot of changes completely unrelated to the commit message in this commit. It would be nice to have them separated, like why is this dependency updated? And why is `babel-preset-2017` introduced?

Reply:
It seems like I committed a bit too much. I thought the es2017 preset for `eslint` could only work in newer versions, but turns out it didn't. The `babel-preset-2017` also didn't belong there. I have cleaned it up accordingly.